`dataSourceNetworkingSecGroupV2Read` crashes with panic

Crash log:
```
panic: Invalid address to set: []string{"description"}

goroutine 33 [running]:
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*ResourceData).Set(0xc00074e300, 0x16f3b8f, 0xb, 0x146f840, 0xc000794180, 0x0, 0x0)
	/home/akachurin/go/pkg/mod/github.com/hashicorp/terraform-plugin-sdk/v2@v2.6.1/helper/schema/resource_data.go:230 +0x371
github.com/opentelekomcloud/terraform-provider-opentelekomcloud/opentelekomcloud/services/vpc.dataSourceNetworkingSecGroupV2Read(0x196d8f8, 0xc00075c7e0, 0xc00074e300, 0x16ded40, 0xc000894c40, 0xc000a50f60, 0xc000af1948, 0x410438)
	/home/akachurin/GolandProjects/terraform-provider-opentelekomcloud/opentelekomcloud/services/vpc/data_source_opentelekomcloud_networking_secgroup_v2.go:79 +0x80f
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).read(0xc0001f56c0, 0x196d888, 0xc000ac05c0, 0xc00074e300, 0x16ded40, 0xc000894c40, 0x0, 0x0, 0x0)
	/home/akachurin/go/pkg/mod/github.com/hashicorp/terraform-plugin-sdk/v2@v2.6.1/helper/schema/resource.go:347 +0x17f
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).ReadDataApply(0xc0001f56c0, 0x196d888, 0xc000ac05c0, 0xc000aae460, 0x16ded40, 0xc000894c40, 0xc000894c40, 0xc000aae460, 0x0, 0x0)
	/home/akachurin/go/pkg/mod/github.com/hashicorp/terraform-plugin-sdk/v2@v2.6.1/helper/schema/resource.go:558 +0xfd
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*GRPCProviderServer).ReadDataSource(0xc000a0c5e8, 0x196d888, 0xc000ac05c0, 0xc000aae340, 0xc000ac05c0, 0x40dca5, 0x15f2cc0)
	/home/akachurin/go/pkg/mod/github.com/hashicorp/terraform-plugin-sdk/v2@v2.6.1/helper/schema/grpc_provider.go:1105 +0x4d6
github.com/hashicorp/terraform-plugin-go/tfprotov5/server.(*server).ReadDataSource(0xc0006a2340, 0x196d930, 0xc000ac05c0, 0xc000734320, 0xc0006a2340, 0xc000aac720, 0xc000ab8ba0)
	/home/akachurin/go/pkg/mod/github.com/hashicorp/terraform-plugin-go@v0.3.0/tfprotov5/server/server.go:247 +0xe5
github.com/hashicorp/terraform-plugin-go/tfprotov5/internal/tfplugin5._Provider_ReadDataSource_Handler(0x16785e0, 0xc0006a2340, 0x196d930, 0xc000aac720, 0xc00075c600, 0x0, 0x196d930, 0xc000aac720, 0xc0008a8310, 0x64)
	/home/akachurin/go/pkg/mod/github.com/hashicorp/terraform-plugin-go@v0.3.0/tfprotov5/internal/tfplugin5/tfplugin5_grpc.pb.go:416 +0x214
google.golang.org/grpc.(*Server).processUnaryRPC(0xc000894a80, 0x1979878, 0xc0008b6000, 0xc00073a100, 0xc0000bed50, 0x21cd830, 0x0, 0x0, 0x0)
	/home/akachurin/go/pkg/mod/google.golang.org/grpc@v1.32.0/server.go:1194 +0x52b
google.golang.org/grpc.(*Server).handleStream(0xc000894a80, 0x1979878, 0xc0008b6000, 0xc00073a100, 0x0)
	/home/akachurin/go/pkg/mod/google.golang.org/grpc@v1.32.0/server.go:1517 +0xd0c
google.golang.org/grpc.(*Server).serveStreams.func1.2(0xc0003a9da0, 0xc000894a80, 0x1979878, 0xc0008b6000, 0xc00073a100)
	/home/akachurin/go/pkg/mod/google.golang.org/grpc@v1.32.0/server.go:859 +0xab
created by google.golang.org/grpc.(*Server).serveStreams.func1
	/home/akachurin/go/pkg/mod/google.golang.org/grpc@v1.32.0/server.go:857 +0x1fd


Process finished with the exit code 1
```

--- a/opentelekomcloud/services/vpc/data_source_opentelekomcloud_networking_secgroup_v2.go
+++ b/opentelekomcloud/services/vpc/data_source_opentelekomcloud_networking_secgroup_v2.go
@@ -4,6 +4,7 @@ import (
 	"context"
 	"log"
 
+	"github.com/hashicorp/go-multierror"
 	"github.com/opentelekomcloud/gophertelekomcloud/openstack/networking/v2/extensions/security/groups"
 
 	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
@@ -36,6 +37,10 @@ func DataSourceNetworkingSecGroupV2() *schema.Resource {
 				Type:     schema.TypeString,
 				Optional: true,
 				ForceNew: true,
+				Computed: true,
+			},
+			"description": {
+				Type:     schema.TypeString,
 				Computed: true,
 			},
 		},
@@ -75,10 +80,15 @@ func dataSourceNetworkingSecGroupV2Read(_ context.Context, d *schema.ResourceDat
 	log.Printf("[DEBUG] Retrieved Security Group %s: %+v", secGroup.ID, secGroup)
 	d.SetId(secGroup.ID)
 
-	d.Set("name", secGroup.Name)
-	d.Set("description", secGroup.Description)
-	d.Set("tenant_id", secGroup.TenantID)
-	d.Set("region", config.GetRegion(d))
+	mErr := multierror.Append(
+		d.Set("name", secGroup.Name),
+		d.Set("description", secGroup.Description),
+		d.Set("tenant_id", secGroup.TenantID),
+		d.Set("region", config.GetRegion(d)),
+	)
+	if err := mErr.ErrorOrNil(); err != nil {
+		return fmterr.Errorf("error setting security group fields: %w", err)
+	}
 
 	return nil
 }

--- a/releasenotes/notes/fix-secgroup-23e41d4ac82f0f34.yaml
+++ b/releasenotes/notes/fix-secgroup-23e41d4ac82f0f34.yaml
@@ -1,0 +1,4 @@
+---
+fixes:
+  - |
+    Fix panic when using `opentelekomcloud_networking_secgroup_v2` data source ([#1176](https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud/pull/1176))

